concent_api/core/tests/utils.py

```diff
@@ -28,10 +31,14 @@ def setUp(self):
         super().setUp()
 
         # Keys
-        (self.PROVIDER_PRIVATE_KEY,  self.PROVIDER_PUBLIC_KEY)    = generate_ecc_key_pair()
-        (self.REQUESTOR_PRIVATE_KEY, self.REQUESTOR_PUBLIC_KEY)   = generate_ecc_key_pair()
-        (self.DIFFERENT_PROVIDER_PRIVATE_KEY, self.DIFFERENT_PROVIDER_PUBLIC_KEY) = generate_ecc_key_pair()
-        (self.DIFFERENT_REQUESTOR_PRIVATE_KEY, self.DIFFERENT_REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
+        (self.PROVIDER_PRIVATE_KEY,                 self.PROVIDER_PUBLIC_KEY)               = generate_ecc_key_pair()
+        (self.REQUESTOR_PRIVATE_KEY,                self.REQUESTOR_PUBLIC_KEY)              = generate_ecc_key_pair()
+        (self.DIFFERENT_PROVIDER_PRIVATE_KEY,       self.DIFFERENT_PROVIDER_PUBLIC_KEY)     = generate_ecc_key_pair()
+        (self.DIFFERENT_REQUESTOR_PRIVATE_KEY,      self.DIFFERENT_REQUESTOR_PUBLIC_KEY)    = generate_ecc_key_pair()
+        (self.PROVIDER_PRIV_ETH_KEY,                self.PROVIDER_ETH_ADDRESS)              = generate_priv_and_address_eth_account()
+        (self.REQUESTOR_PRIV_ETH_KEY,               self.REQUESTOR_ETH_ADDRESS)             = generate_priv_and_address_eth_account()
+        (self.DIFFERENT_PROVIDER_PRIV_ETH_KEY,      self.DIFFERENT_PROVIDER_ETH_ADDRESS)    = generate_priv_and_address_eth_account()
```

Comment:
I don't see those two (DIFFERENT_PROVIDER_*_ETH_KEY and, DIFFERENT_REQUESTOR_*_ETH_KEY) used anywhere, if so they are not need at the moment.